lib/rules/time-min-milliseconds/index.js

```diff
@@ -118,27 +107,27 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		/**
-		 * @param {string} time
+		 * @param {import('postcss-value-parser').Dimension | {unit: null, number: null}} valueNode
 		 * @returns {boolean}
 		 */
-		function isAcceptableTime(time) {
-			const parsedTime = valueParser.unit(time);
+		function isAcceptableTime(valueNode) {
+			const { unit, number } = valueNode;
 
-			if (!parsedTime) return true;
+			if (unit === null || number === null) return true;
 
-			const numTime = Number(parsedTime.number);
+			const numTime = Number(number);
 
 			if (numTime <= 0) {
 				return true;
 			}
 
-			const unit = parsedTime.unit.toLowerCase();
+			const _unit = unit.toLowerCase();
```

Comment:
[suggest] Using the `_` prefix for variables is rare in our code base. How about renaming the variable like this?

```suggestion
			const timeUnit = unit.toLowerCase();
```